examples/scitail.py

```diff
@@ -0,0 +1,172 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and Simon Ott (nomisto)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The SciTail dataset is an entailment dataset created from multiple-choice science exams and
+web sentences. Each question and the correct answer choice are converted into an assertive
+statement to form the hypothesis. We use information retrieval to obtain relevant text from
+a large text corpus of web sentences, and use these sentences as a premise P. We crowdsource
+the annotation of such premise-hypothesis pair as supports (entails) or not (neutral), in order
+to create the SciTail dataset. The dataset contains 27,026 examples with 10,101 examples with
+entails label and 16,925 examples with neutral label.
+"""
+
+import os
+from dataclasses import dataclass
+
+import datasets
+import pandas as pd
+
+_CITATION = """\
+@inproceedings{scitail,
+    author = {Tushar Khot and Ashish Sabharwal and Peter Clark},
+    booktitle = {AAAI}
+    title = {SciTail: A Textual Entailment Dataset from Science Question Answering},
+    year = {2018}
+}
+"""
+
+_DATASETNAME = "scitail"
+
+_DESCRIPTION = """\
+The SciTail dataset is an entailment dataset created from multiple-choice science exams and
+web sentences. Each question and the correct answer choice are converted into an assertive
+statement to form the hypothesis. We use information retrieval to obtain relevant text from
+a large text corpus of web sentences, and use these sentences as a premise P. We crowdsource
+the annotation of such premise-hypothesis pair as supports (entails) or not (neutral), in order
+to create the SciTail dataset. The dataset contains 27,026 examples with 10,101 examples with
+entails label and 16,925 examples with neutral label.
+"""
+
+_HOMEPAGE = "https://allenai.org/data/scitail"
+
+_LICENSE = "Apache License 2.0"
+
+_URLS = {
+    _DATASETNAME: "https://ai2-public-datasets.s3.amazonaws.com/scitail/SciTailV1.1.zip",
+}
+
+_SUPPORTED_TASKS = ["entailment"]
```

Comment:
fresh of the presses, new and updated task names :) 
```suggestion
_SUPPORTED_TASKS = ["TE"]
```